operators/multiclusterobservability/pkg/util/managedclusteraddon.go

```diff
@@ -93,6 +93,8 @@ func CreateManagedClusterAddonCR(c client.Client, namespace, labelKey, labelValu
 		}
 
 		// got the created managedclusteraddon just now, uopdating its status
+		// TODO(saswatamcode): Remove deprecated field
+		//nolint:staticcheck
```

Comment:
Why the `nolint` clause?

Reply:
The reason is mentioned in the TODO. That field is deprecated, so probably needs some logical changes. For now it would fail staticcheck, due to it being marked deprecated.